examples/advanced-component-usage/sortable_table.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+"""
+A simple application demonstrating how to build a sortable dbc.Table using sorttable.js
+(https://www.kryogenix.org/code/browser/sorttable/), a small JS library to make an html table sortable.
+"""
+
+import dash
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, MATCH, Output, State
+
+DUMMY_STORE_TYPE = 'dummy-store'
+SORTABLE_TABLE_TYPE = 'sortable-table'
+COMPONENT_ID_INDEX = 1
+
+app = dash.Dash(__name__,
+                assets_folder='sorttable_assets',
+                external_stylesheets=[dbc.themes.BOOTSTRAP],
+                suppress_callback_exceptions=True)
+
+
+def create_table(num_rows):
+    accordion = html.Div(
+        [
+            dbc.Table(
+                [
+                    html.Thead(
+                        html.Tr([
+                            html.Th('Name'),
+                            html.Th('Number')
+                        ])
```

Comment:
Can we add `cursor:pointer` to these to make it clearer they are clickable?

Reply:
Again thanks for the tip, I was looking how to do it but had no time to really invest in this as I was on hurry with developping a production app where it was a detail.